### ISSUE TITLE
[OUDS] Hidden focus outline on breadcrumb links

### ISSUE THREAD

**Issue by MaxLardenois** [Orange-OpenSource/Orange-Boosted-Bootstrap#3020]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The focus outline on breadcrumb item is partially hidden.

We have 

<img width="319" alt="Image" src="https://github.com/user-attachments/assets/e3045bf2-a362-4c0b-89a3-90072c4bf04f" />

Where we should have 

<img width="315" alt="Image" src="https://github.com/user-attachments/assets/4743b487-248a-4dd2-bd58-6b4f33e92281" />

This is due to a regression [in this commit](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/627094b390da1630591432413fd242138dce8d48#diff-4219169e92f41a59d6f67d6e226b02f0dcc0a740f4c8eb48b16dcd858ffa47e2R24)
The overflow: hidden added to be somewhat compatible with Bootstrap DOM for Breadcrumb hide parts of the links outline.

### Reduced test cases

Tap on tab key on a Breadcrumb

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

v0.4.1
